node/src/blockchain/blockchain_bridge.rs

```diff
@@ -535,6 +537,10 @@ struct PendingTxInfo {
     when_sent: SystemTime,
 }
 
+pub fn increase_gas_price_by_margin(gas_price: u128, chain: Chain) -> u128 {
+    (gas_price * (100 + chain.rec().gas_price_default_margin_percents as u128)) / 100
```

Comment:
In case we want the user to give the ability to customise the margin, it would be better if we use the same margin for all the blockchains, and it will eliminate the need to fetching it from the `chain` record, since it'll become a constant for now.